docs/templates/language-reference.md

```diff
@@ -49,10 +49,30 @@ Depending on the language, a brief description of the recommended way to handle
 If available, include examples for connecting in an app
 (taken from https://examples.docs.platform.sh/).
 
-### Configuration reader
+{{< codetabs >}}
 
-If available, include the configuration reader available for the given language
-(for example, for [Python](https://github.com/platformsh/config-reader-python)).
+---
+title=<EXAMPLE_1>
```

Comment:
:lady_beetle: This isn't really descriptive as a label. The number isn't as helpful as it could be.

Also, it obscures what should be placed there, which is not just an example.